libs/sync/src/lib/sync-details-page/sync-modal/sync-modal.component.scss

```diff
@@ -0,0 +1,9 @@
+@import 'apps/dashboard/src/variables';
+
+.actions {
+  display: flex;
```

Comment:
this would not be needed by using the `pa-modal-footer` directive (or a confirm dialog directly)